packages/plugin-go/tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "noUnusedParameters": true,
     "outDir": "dist",
     "strict": true,
-    "target": "esnext"
+    "target": "esnext",
+    "resolveJsonModule": true
```

Comment:
@TooTallNate I'm not sure if you were the one who told me about it before, but was there a reason for why we didn't add `resolveJsonModule` to the `vercel` repo before, did it have some side effects?

Reply:
No not that I'm aware of. I think we just tent to not add this option until/unless a `.json` file actually is being imported.

Reply:
It only works when `module: "commonjs"` so I think we didn't want to use it if we ever moved our code to real modules ESM. Hopefully TS would error but not sure.

The other alternative is to use an actual `require()` so its clear it only works with require and not import.